src/search_request.c

```diff
@@ -10,6 +10,34 @@
 #include "rmalloc.h"
 #include <sys/param.h>
 
+/**
+ * Gets a variable list of arguments to a given keyword.
+ * The number of variable arguments are found in `outlen`.
+ * If offset is specified
+ */
+static RedisModuleString **getLengthArgs(const char *keyword, size_t *outlen,
```

Comment:
Actually not a must for merging this, but I want this in RMUTIL, I need it in a another module